Dockerfile-public-image

```diff
@@ -0,0 +1,53 @@
+FROM registry.fedoraproject.org/fedora:36
```

Comment:
Ok, so this was caused by lack of my knowledge, I thought that for the UBI you need subscription too, so in that case, we can drop this change and in the end only the redis and postgres images would need to be different.

Also there were some additional rpm packages installed so the pip install wouldn't complain, but it doesn't matter anyway, since we can easily drop this change.